src/openapi/peridio-admin-openapi.yaml

```diff
@@ -4534,6 +4534,48 @@ components:
           $ref: "#/components/schemas/binary-signature-signature"
         signing_key_prn:
           $ref: "#/components/schemas/prn"
+
+    tunnel:
+      type: object
+      properties:
+        device_proxy_ip_address:
+          type: string
+          examples: [10.0.1.1]
+          description: The wireguard interface IP address for the device.
+        device_proxy_port:
+          type: number
+          examples: [47539]
+          description: The port on which the device is listening for wireguard UDP traffic.
+        device_tunnel_port:
+          type: number
+          examples: [22]
+          description: The port on which the device is listening for wireguard UDP traffic.
+        device_public_key:
```

Comment:
Needs a type and description. This is causing consumers of this schema to choke right now without it.